### PR TITLE
fix(Table): update sticky css props

### LINKED ISSUE #8829: Table - Add support for sticky column on right
**Is this a new component or an extension of an existing one?**
What is the existing component, if any?
Table

**Describe the feature**
 Follow up for core issue https://github.com/patternfly/patternfly/issues/5391

### PR DESCRIPTION
**Pull request by kmcfaul**
**What**: Closes #8829

Updated table sticky feature to use the updated CSS variables, and set the default of the offset variables to 0. This means that both left and right will be set by default, but it shouldn't interfere with the sticky alignment. The alternative is to require that `stickyRightOffset` and `stickyLeftOffset` be set even for the first/last sticky columns with a 0 value.

### CODE REVIEW

File: packages/react-table/src/components/Table/examples/Table.md [hunk 1]
```diff
@@ -332,17 +332,10 @@ To make a column sticky, wrap `Table` with `InnerScrollContainer` and add the fo
...
-- add hasRightBorder to the rightmost sticky column
-- add stickyLeftOffset to each sticky column after the first, with a value that equals the combined width - set by stickyMindWidth - of the previous sticky columns
-
-To prevent the default text wrapping behavior and allow horizontal scrolling, all Th or Td cells should also have the modifier="nowrap" property.
-
 - wrap `Table` with `InnerScrollContainer`
 - add `isStickyColumn` to all columns that should be sticky
 - add `hasRightBorder` to the rightmost sticky column
-- add `stickyLeftOffset` to each sticky column after the first, with a value that equals the combined width - set by `stickyMindWidth` - of the previous sticky columns
+- add `stickyLeftOffset` to each sticky column with a value that equals the combined width - set by `stickyMindWidth` - of the previous sticky columns, or 0 for the leftmost column
```
Comment (thatblindgeye) [inline packages/react-table/src/components/Table/examples/Table.md:338]: If we keep the default of the CSS variables as 0, we could update the last bit to something like, "...of the previous sticky column. The leftmost column should have a value of `0`, which is the default value of this property." Or something to convey that they may not need to pass this property for the leftmost column

File: packages/react-table/src/components/Table/examples/Table.md [hunk 2]
```diff
@@ -323,7 +323,7 @@ There are a few ways this can be handled:
 
 ### Sticky column
 
-To make a column sticky, wrap `Table` with `InnerScrollContainer` and add the following properties to the `Th` or `Td` that should be sticky: `isStickyColumn` and `hasRightBorder`. To prevent the default text wrapping behavior and allow horizontal scrolling, all `Th` or `Td` cells should also have the `modifier="nowrap"` property. To set the minimum width of the sticky column, use the `stickyMinWidth` property.
+To make a column sticky, wrap `Table` with `InnerScrollContainer` and add the following properties to the `Th` or `Td` that should be sticky: `isStickyColumn`, `stickyLeftOffset`, and `hasRightBorder`. For right-aligned sticky columns, add `stickyRightOffset` and `hasLeftBorder` instead. To prevent the default text wrapping behavior and allow horizontal scrolling, all `Th` or `Td` cells should also have the `modifier="nowrap"` property. To set the minimum width of the sticky column, use the `stickyMinWidth` property.
```
[the remark this answers is not in the document]
Comment (kmcfaul) [reply packages/react-table/src/components/Table/examples/Table.md]: Updated the wording a little bit, moving the offset props further along. Does that read better or should I just remove that blurb?

**Review by nicolethoen — APPROVED** [review]

**Review by thatblindgeye — APPROVED** [review]
Looks good!

**Comment by kmcfaul** [comment]
This PR also addresses https://github.com/patternfly/patternfly-react/issues/8776

**Review by thatblindgeye — APPROVED** [review]
CSS-wise this looks good

**Pull request merged**